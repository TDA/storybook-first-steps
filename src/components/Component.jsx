import React from "react";

const Component = ({value}) => {
  return <div>{value}</div>;
};
export default Component;