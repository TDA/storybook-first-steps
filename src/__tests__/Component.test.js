import React from 'react';
import renderer from 'react-test-renderer';
import Component from "../components/Component";
import {shallow} from "enzyme";

test('Component displays values', () => {
  const c = renderer.create(<Component value={'ram'}/> );

  expect(c).toMatchSnapshot();
});

test('Component displays values when shallow rendered', () => {
  shallow(<Component value={'sai'}/> );
});
