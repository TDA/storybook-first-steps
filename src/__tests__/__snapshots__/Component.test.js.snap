// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component displays values 1`] = `
<div>
  ram
</div>
`;
